File: cli/testdata/php/gost_usage.php
<?php
// GOST R 34.11 / Streebog hash via PHP hash() extension
// EXPECTED: GOST3411 | hash | | info |
function gostHash($data) {
    return hash("gost-crypto", $data);
}

function streebogHash($data) {
    return hash("streebog256", $data);
}

File: cli/testdata/rust/gost_usage.rs
// GOST R 34.11 / Streebog hash via RustCrypto streebog crate
// EXPECTED: GOST3411 | hash | | info |
use streebog::Streebog256;
use streebog::Digest;

fn hash_data(data: &[u8]) -> Vec<u8> {
    let mut hasher = Streebog256::new();
    hasher.update(data);
    hasher.finalize().to_vec()
}
